### PR TITLE
new test of empirical incubation dist

### PR DESCRIPTION
**Pull request by ffinger**
Added new test as discussed.

### CODE REVIEW

File: tests/testthat/test_estimate_incubation.R [hunk 1]
```diff
@@ -22,7 +22,25 @@ exposures_two <- vector(mode = "list", length = nrow(ll))
 for (i in seq_along(exposures)) {
   exposures_two[[i]] <- sample(seq(ll$exposure_start[i], ll$exposure_end[i], by = "1 day"))
 }
-ll$boogaloo <- exposures_two 
+ll$boogaloo <- exposures_two
+
+#creating a simple line list with known incubation period dist:
+ll2 <- data.frame(
+  patient_id = 1:4,
+  onset = lubridate::as_date(c("2018-1-15", "2018-1-20", "2018-1-23", "2018-1-24")
```
Comment (zkamvar) [inline tests/testthat/test_estimate_incubation.R]: please use:

```r
onset = as.Date(c("2018-01-15", "2018-01-20", "2018-01-23", "2018-01-24"))
```
```

**Comment by zkamvar** [comment]
Thanks! This is great ^_^ 

The only suggestion I have is to use R's default `as.Date()` instead of `lubridate::as_date()` since epitrix does not yet depend on lubridate for anything (though linelist will).

File: tests/testthat/test_estimate_incubation.R [hunk 2]
```diff
@@ -75,6 +93,14 @@ test_that("empirical incubation period distribution matches reference", {
 })
 
 
+test_that("empirical incubation period distribution matches dist reference computed by hand", {
+  skip_on_cran()
```
Comment (ffinger) [inline tests/testthat/test_estimate_incubation.R]: I dont know which tests need `skip_on_cran()`, will let you decide if to leave it in or not.

Comment (zkamvar) [reply tests/testthat/test_estimate_incubation.R]: I think this particular test is small enough to let past CRAN :)

IMO: the criteria I have for letting a test on CRAN:

1. The test runs fast (e.g. about a millisecond)
1. If something goes wrong with the test, I can identify what it is relatively quickly
1. I know without a doubt that I control the input and output data

**Comment by zkamvar** [comment]
Awesome! Thank you, @ffinger

**Pull request merged**